core/templates/dev/head/pages/dashboard/Dashboard.js

```diff
@@ -205,7 +225,11 @@ oppia.controller('Dashboard', [
         } else {
           $scope.activeTab = 'myExplorations';
         }
-        $rootScope.loadingMessage = '';
+        if ($location.path() === '/back') {
```

Comment:
Thanks -- removing /back (while still keeping things working) sounds like a move in the right direction!